### PR TITLE
Adding endpoint for container

### PR DESCRIPTION
**Pull request by versatil56**

### CODE REVIEW

File: test/uk/gov/hmrc/digitalengagementplatformpartials/views/NuanceTagElementSpec.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            }
+
+            "include div tag element with custom ID" in {
+                val loadedView = view("myTest");
+                val queriableView : Document = Jsoup.parse(loadedView.toString())
+
+                queriableView.getElementById("myTest") should not be null
+            }
+        }
+    }
```
Comment (tonymyers) [inline test/uk/gov/hmrc/digitalengagementplatformpartials/views/NuanceTagElementSpec.scala:47]: Do we need to test what happens if div tag is not loaded?

**Review by tonymyers — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/digitalengagementplatformpartials/views/NuanceTagElement.scala.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.digitalengagementplatformpartials.config.AppConfig
+
+@this(appConfig: AppConfig)
+
+@(id : String = "HMRC_Fixed_1")
+
+<div id="@id"></div>
```
[the remark this answers is not in the document]
Comment (versatil56) [reply app/uk/gov/hmrc/digitalengagementplatformpartials/views/NuanceTagElement.scala.html:23]: No, you will have to make a request with the appropriate id in it. If you look at app.routes you will see that we take in an id parameter.

File: test/uk/gov/hmrc/digitalengagementplatformpartials/views/NuanceTagElementSpec.scala (hunk 1 above)
Comment (versatil56) [reply test/uk/gov/hmrc/digitalengagementplatformpartials/views/NuanceTagElementSpec.scala:47]: I dont think this is a possible code path?

Comment (tonymyers) [reply test/uk/gov/hmrc/digitalengagementplatformpartials/views/NuanceTagElementSpec.scala:47]: That, makes sense, thanks.

**Review by tonymyers — APPROVED** [review]

**Pull request merged**